hazelcast/src/main/java/com/hazelcast/internal/serialization/SerializationService.java

```diff
@@ -42,6 +42,8 @@
 
     <T> T toObject(Object data);
 
+    <T> T bytesToObject(byte[] bytes, int dataOffset);
```

Comment:
So we always rely on the fact that object is totally presented in byte[] array ?

What is the goal of dataOffset in this case ?